.github/CODEOWNERS

```diff
@@ -0,0 +1,42 @@
+* @bentoml/bentoml-reviewers
+
+# User-facing API changes:
+/bentoml/*.py @ssheng @parano @bentoml/bentoml-reviewers
+
+# BentoML Internals
+/bentoml/_internal/bento/docker/ @aarnphm @bentoml/bentoml-reviewers
+/bentoml/_internal/bento/ @sauyon @parano @aarnphm @bentoml/bentoml-reviewers
+/bentoml/_internal/server/ @bojiang @ssheng @bentoml/bentoml-reviewers
+/bentoml/_internal/frameworks/ @larme @bentoml/bentoml-reviewers
+/bentoml/_internal/io_descriptors/ @aarnphm @sauyon @bentoml/bentoml-reviewers
+/bentoml/_internal/runner/ @bojiang @sauyon @bentoml/bentoml-reviewers
+/bentoml/_internal/cli/ @aarnphm @larme @bentoml/bentoml-reviewers
+/bentoml/_internal/yatai* @yetone @bentoml/bentoml-reviewers
+/bentoml/_internal/service/ @bojiang @bentoml/bentoml-reviewers
+/bentoml/_internal/utils/analytics @ssheng @parano  @bentoml/bentoml-reviewers
+/bentoml/_internal/configuration/ @ssheng @bentoml/bentoml-reviewers
+/bentoml/_internal/models/ @sauyon @bentoml/bentoml-reviewers
+/bentoml/_internal/marshal/ @bojiang @bentoml/bentoml-reviewers
+/bentoml/_internal/testing/ @bojiang @bentoml/bentoml-reviewers
+/bentoml/_internal/external_typing/ @sauyon @bentoml/bentoml-reviewers
+/bentoml/_internal/*.py @sauyon @bentoml/bentoml-reviewers
```

Comment:
Do you think this can be simplified to just the team, `@bentoml/bentoml-reviewers`?